pkgs/development/tools/misc/c2ffi/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, cmake
+, llvmPackages_11
+, unstableGitUpdater
+}:
+
+let
+  c2ffiBranch = "llvm-11.0.0";
+  llvmPackages = llvmPackages_11;
+in
+
+llvmPackages.stdenv.mkDerivation {
+  pname = "c2ffi";
+  version = c2ffiBranch + "-unstable-2021-04-15";
```

Comment:
```suggestion
  pname = "c2ffi-${c2ffiBranch}";
  version = "unstable-2021-04-15";
```

Reply:
+1. The unstable updater won't work otherwise